_ap/17/index.adoc

```diff
@@ -0,0 +1,58 @@
+---
+num: 17
+tags:
+  - "tls"
+  - "certificates"
+  - "dns"
+  - "Letsencrypt"
+title: "Automatic management of certificates per managed instance using Letsencrypt"
```

Comment:
How much of this can/is/make sense to include in factorised fleet manager? 
Is it possible to make it "almost" for free for new services?

Reply:
There are ACME libraries / operators that can be used. That's what I'll advise new services using depending on their deployment topology. 
Also the way this is implemented in kafka services, can be extracted onto its own SDK as part of the factorized fleet management effort. The SDK part is the second phase of that effort and involves multiple part of the the KFM codebase.